libraries/AP_BattMonitor/AP_BattMonitor_Backend.cpp

```diff
@@ -17,6 +17,7 @@
 #include <AP_HAL/AP_HAL.h>
 #include "AP_BattMonitor.h"
 #include "AP_BattMonitor_Backend.h"
+#include <GCS_MAVLink/GCS.h>
```

Comment:
Why was this needed? Nothing jumps out at me as a missing feature.

Reply:
Good question. Probably by accident. Deleting.